Makefile

```diff
@@ -20,45 +20,45 @@ clean-c2pa-env: clean
 install-deps:
 	python3 -m pip install -r requirements.txt
 	python3 -m pip install -r requirements-dev.txt
-	pip install -e .
 
 # Installs the package in development mode
 build-python:
-	pip install -e .
+	python3 -m pip install -e .
 
 # Performs a complete rebuild of the development environment
 rebuild: clean-c2pa-env install-deps download-native-artifacts build-python
 	@echo "Development rebuild done"
 
 run-examples:
 	python3 ./examples/sign.py
+	python3 ./examples/sign_info.py
 	python3 ./examples/training.py
 	rm -rf output/
 
 # Runs the examples, then the unit tests
 test:
-	python3 ./tests/test_unit_tests.py
-	python3 ./tests/test_unit_tests_threaded.py
+	PYTHONWARNINGS=ignore::DeprecationWarning python3 ./tests/test_unit_tests.py
```

Comment:
What is causing the deprecation errors here? can we update them to avoid this?

Reply:
It's my tests on the legacy API that log deprecation warnings.